From 51710fd7333aee32a7989d2a588fd5afea873b23 Mon Sep 17 00:00:00 2001
From: Larry Finger <Larry.Finger@lwfinger.net>
Date: Tue, 1 Apr 2014 12:07:55 -0500
Subject: [PATCH 460/470] rtl8192du: Remove SUCCESS and FAIL

These two macros are not used. In addition, their definitions
conflict with _SUCCESS and _FAIL.

Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
---
 include/basic_types.h | 4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/staging/rtl8192du/include/basic_types.h
+++ b/drivers/staging/rtl8192du/include/basic_types.h
@@ -18,10 +18,6 @@
 
 #include <autoconf.h>
 
-
-#define SUCCESS	0
-#define FAIL	(-1)
-
 #include <linux/types.h>
 
 #define FIELD_OFFSET(s,field)	((__kernel_ssize_t)&((s*)(0))->field)
